paper/paper.md

```diff
@@ -0,0 +1,333 @@
+progress.
+We hosted the curriculum in a public GitHub repository
+(https://github.com/umcarpentries/intro-curriculum-r) to facilitate
+collaborative work and peer review using issues, branches, and pull requests.
+Under this model, a team member creates or edits content in a new branch to
+resolve an issue,
+then creates a pull request and asks for review from another team member,
+who finally merges the changes into the default branch.
+GitHub pages automatically uses the default branch to build a website which
+hosts the polished curriculum (https://umcarpentries.org/intro-curriculum-r/).
+This model ensures that at least two pairs of eyes view the changes before they
+can be included in the curriculum, which reduces mistakes and contributes to
+better quality content.
+
+![The curriculum development framework. Created with BioRender.com.](development-framework.png)
+
+Following the sprint, contributors finalized edits and continued to review each
+others' pull requests to complete the alpha version of our curriculum.
+Next, we hosted a workshop for instructors to pilot the curriculum.
+We collected feedback from the learners and instructors at the end of the pilot
+workshop, then held a smaller half-day sprint to revise the curriculum based on
+the feedback.
+Community members continue to create issues, make edits, and review pull
+requests for ongoing refinement of the curriculum.
+We have plans to teach more workshops with instructors who did not develop the
+curriculum to gather even more feedback.
+
+# Curriculum
+
+Our curriculum is tailored to people with no prior coding experience who want to
+learn how to use programming to perform data analysis (Figure 2).
+We aim to teach our learners not only the basics of performing data analysis,
+but also to provide them with a framework to perform rigorous reproducible
+research that can easily be shared with others.
+
+![Curriculum overview. Created with BioRender.com.](curriculum-overview.png)
+
+## Learning Objectives
+
+The key learning objectives for our curriculum are:
+
+1. Create clear and informative data visualizations in R, starting with messy data.
+1. Perform version control using the Unix shell and git.
+1. Create reproducible reports using R markdown.
+1. Share code with others on GitHub.
+
+We believe these skills provide a solid foundation from which learners can teach
+themselves any additional coding skills that they will need in the future.
+
+## Course Content
+
+Our curriculum consists of modules that cover setup, data analysis and
+visualization, version control, sharing code, and writing reports.
+We also provide a goal for the workshop: to create a fictitious report for the
+United Nations on the relationship between gross domestic product (GDP), life
+expectancy, and CO~2~ emissions.
+The R programming lessons take a "tidyverse first" approach
+[@robinson_teach_2017] so learners can meet this goal.
```

Comment:
I am not sure i understand how the tidyverse first approach helps learners achieve this goal. What is unique about this approach that enables this? Perhaps it would be good to explain this at least in a half sentence instead of just citing the paper that outlines the approach?

Reply:
cc @zenalapp @sklucas, anyone want to take a stab at this?

Reply:
(if not, I'll probably be able to get to it next week)

Reply:
@sklucas if you could that'd be great!

Reply:
```suggestion
[@robinson_teach_2017] to quickly teach learners powerful tools for plotting and data analysis.
```

@kelly-sovacool @sklucas here's one possibility. not sure 'quickly' is quite the right word, but can't think of anything better right now

Reply:
Pat's suggestions: Effectively or efficiently

```suggestion
[@robinson_teach_2017] to effectively and efficiently teach learners powerful tools for plotting and data analysis.
```